### ISSUE TITLE
[Crash] Fury applied to the product of a Call for Backup card will crash the game

### ISSUE THREAD

**Issue by hunky-dory** [TerryCavanagh/diceydungeons.com#1114]
![gif](https://giant.gfycat.com/NastyUnsungKid.gif)
https://giant.gfycat.com/NastyUnsungKid

Turn 1: Apply fury to a regular card = no problem
Turn 2: Apply fury to the same card delivered by the $replacewith()$ method = crash

(Reequipnext and Recycle work as expected in the same scenario, albeit with separate issue #1108)
Here's the error log. No additional information appears in the console.
```
--------------------------------------
filename:	diceydungeons
package:	com.terrycavanaghgames.diceydungeons
version:	0.17.2 / 0.7.0 (mod API)
sess. ID:	dicey_dungeons_2019-06-08_11'33'21
started:	2019-06-08 11:33:21
--------------------------------------
crashed:	2019-06-08 11:35:12
duration:	00:01:51
error:		Null Object Reference
stack:
states.Combat.repeatlastaction (states/Combat.hx line 1345)
elements.Equipment.doequipmentaction (elements/Equipment.hx line 1876)
Reflect.callMethod (/usr/local/lib/haxe/std/cpp/_std/Reflect.hx line 55)
motion.actuators.GenericActuator.complete (motion/actuators/GenericActuator.hx line 155)
motion.actuators.SimpleActuator.update (motion/actuators/SimpleActuator.hx line 529)
motion.actuators.SimpleActuator.stage_onEnterFrame (motion/actuators/SimpleActuator.hx line 608)
openfl.events.EventDispatcher.__dispatchEvent (openfl/events/EventDispatcher.hx line 443)
openfl.display.DisplayObject.__dispatch (openfl/display/DisplayObject.hx line 1236)
```

**Comment by hunky-dory** [comment]
Looks like furying a PU Backup? Thief card product also produces this crash (item 3):
![Screenshot 2019-08-15 at 10 00 02](https://user-images.githubusercontent.com/48322734/63084613-7fd6ac00-bf43-11e9-9ef0-d8922804f5f9.png)

**Comment by TerryCavanagh** [comment]
I did a quick check to confirm, and it looks like this was fixed with the other CfB fixes in v1.9! Closing.
